catkin_tools/verbs/catkin_build/job.py

```diff
@@ -139,6 +141,32 @@ def __init__(self, package, package_path, context, force_cmake):
         Job.__init__(self, package, package_path, context, force_cmake)
         self.commands = self.get_commands()
 
+    def get_multiarch(self):
+        if not sys.platform.lower().startswith('linux'):
+            return ''
+        # this function returns the suffix for lib directories on supported systems or an empty string
+        # it uses two step approach to look for multiarch: first run gcc -print-multiarch and if
+        # failed try to run dpkg-architecture
+        throws = False
+        try:
+            p = subprocess.Popen(
+                ['gcc', '-print-multiarch'],
+                stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+            out, err = p.communicate()
+        except:
+            throws = True
+        if throws or p.returncode != 0:
+            try:
+                out, err = subprocess.Popen(
+                    ['dpkg-architecture', '-qDEB_HOST_MULTIARCH'],
+                    stdout=subprocess.PIPE, stderr=subprocess.PIPE).communicate()
+            except:
+                return ""
```

Comment:
Nit pick: Can you either make this `return ''` or update the above `return ''` to also be `return ""`?